seaborn/matrix.py

```diff
@@ -1275,32 +1287,32 @@ def clustermap(
         Either 0 (rows) or 1 (columns). Whether or not to standardize that
         dimension, meaning for each row or column, subtract the minimum and
         divide each by its maximum.
-    figsize : (width, height), optional
+    figsize : tuple of (width, height), optional
         Overall size of the figure.
     cbar_kws : dict, optional
-        Keyword arguments to pass to ``cbar_kws`` in :func:`heatmap`, e.g. to
```

Comment:
Double backticks are needed to get code-formatted text in restructured text (i.e. for the sphinx docs)

Reply:
My rationale of using single backticks was adhering to [numpydoc's guidelines](https://numpydoc.readthedocs.io/en/latest/format.html#sections) that prefer single backticks for keyword arguments mentioned within docstrings. Do you prefer such mentions to be code-formatted text?

Reply:
Hm yes ok. Apparently the interpretation of single backticks by sphinx can be [configured](https://github.com/sympy/sympy/issues/13519), so I guess that's fine as long as their role is set to code in the config file.

Reply:
Thanks, with [https://github.com/mwaskom/seaborn/pull/2313/commits/b35754cb6f0ec5eb3357293fd07e48f2c55acfea](https://github.com/mwaskom/seaborn/pull/2313/commits/b35754cb6f0ec5eb3357293fd07e48f2c55acfea) this becomes the default formatting by sphinx.